src/test/java/core/basesyntax/strategy/BalanceOperationTest.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.strategy;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.OperationType;
+import core.basesyntax.storage.Storage;
+import core.basesyntax.strategy.impl.BalanceOperation;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class BalanceOperationTest {
+    private static Operation balanceOperation;
+
+    @BeforeAll
+    static void setUp() {
```

Comment:
It seems that setUp() method uses with @BeforeEach annotation. And for @BeforeAll it's better to use beforeAll()